site/content/docs/0.0/utilities/api.md

```diff
@@ -258,7 +258,7 @@ Output:
 .opacity-100-hover:hover { opacity: 1 !important; }
 ```
 
-<!--### Responsive
+### Responsive
 
 Add the `responsive` boolean to generate responsive utilities (e.g., `.opacity-md-25`) across [all breakpoints]({{< docsref "/layout/breakpoints" >}}).
```

Comment:
This example only works when `$enable-bootstrap-compatibility: true`. Shouldn't we adapt with OUDS opacity utilities instead?

Reply:
The example with opaque etc. would be more difficult to understand for the readers, so let's keep this simple one.